tests/api/v1/endpoints/test_drp_endpoints.py

```diff
@@ -348,3 +350,80 @@ def test_get_privacy_request_with_drp_action(
             privacy_request_with_drp_action.requested_at.isoformat()
             == response.json()["received_at"]
         )
+
+
+class TestGetDrpDataRights:
+    """
+    Tests for the endpoint to retrieve DRP data rights.
+    """
+
+    @pytest.fixture(scope="function")
+    def url_for_data_rights(self) -> str:
+        return V1_URL_PREFIX + DRP_DATA_RIGHTS
```

Comment:
I'm not sure I see a good way around this in the near-term — this may be confusing because our API is versioned differently to the DRP version.